src/Microsoft.DotNet.XliffTasks/build/Microsoft.DotNet.XliffTasks.targets

```
@@ -2,8 +2,7 @@
 <Project>
 
   <PropertyGroup>
-    <XliffTasksDirectory Condition="'$(XliffTasksDirectory)' == '' and '$(MSBuildRuntimeType)' == 'Core'">$(MSBuildThisFileDirectory)..\tools\net9.0\</XliffTasksDirectory>
-    <XliffTasksDirectory Condition="'$(XliffTasksDirectory)' == '' and '$(MSBuildRuntimeType)' != 'Core'">$(MSBuildThisFileDirectory)..\tools\net472\</XliffTasksDirectory>
+    <XliffTasksDirectory Condition="'$(XliffTasksDirectory)' == ''">$(MSBuildThisFileDirectory)..\tools\$(MSBuildRuntimeType)\</XliffTasksDirectory>
```

I think it would be better (safer) to not rely on $(MSBuildRuntimeType) here. Also, I would use other terms than `Full` and `Core` as those don't reflect the current framework names anymore.

```suggestion
    <XliffTasksDirectory Condition="'$(XliffTasksDirectory)' == '' and '$(MSBuildRuntimeType)' == 'Core'">$(MSBuildThisFileDirectory)..\tools\net\</XliffTasksDirectory>
    <XliffTasksDirectory Condition="'$(XliffTasksDirectory)' == '' and '$(MSBuildRuntimeType)' != 'Core'">$(MSBuildThisFileDirectory)..\tools\netframework\</XliffTasksDirectory>
```